packages/govern-console/src/components/Wallet/Wallet.tsx

```diff
@@ -61,34 +26,34 @@ const Wallet = ({}) => {
       }
     } else if (status === 'connected') {
       setNetworkStatus('connected');
-      enqueueSnackbar('Your wallet is successfully connected.', {
+      toast('Your wallet is successfully connected.', {
         variant: 'success',
       });
     } else {
       setNetworkStatus('disconnected');
     }
-  }, [status, chainId, enqueueSnackbar, error]);
+  }, [status, chainId, toast, error]);
 
   useEffect(() => {
     if (status === 'disconnected') {
       return;
     }
     if (error) {
       if (error.message.includes('Unsupported chain')) {
-        enqueueSnackbar('Please select the correct chain in your wallet.', {
+        toast('Please select the correct chain in your wallet.', {
           variant: 'error',
         });
       } else if (error.message.includes('window.ethereum')) {
-        enqueueSnackbar('Please install a wallet.', {
+        toast('Please install a wallet.', {
           variant: 'error',
         });
       } else {
-        enqueueSnackbar(error.message, {
+        toast(error.message, {
           variant: 'error',
         });
       }
     }
-  }, [status, error, enqueueSnackbar]);
+  }, [status, error, toast]);
 
```

Comment:
will create a separate PR to replace all enqueueSnackbar with toast for consistent look.